README.md

```diff
@@ -44,7 +45,40 @@ for hero in marvel_heroes.run(connection):
 ```
 
 ## Run tests
-To ensure python driver works with python 2.7, 3.4, 3.5, 3.6 we are using `tox` and `pytest`. For testing (depending on what you would like to test) you can run `tox` or `pytest`.
+In the `Makefile` you can find three different test commands: `test-unit`, `test-integration` and `test-remote`. As RebirthDB dropping the support of Windows, we would like to ensure that those of us who are using Windows for development can still contribute. Because of this, we support running integration tests against Digital Ocean Droplets as well.
```

Comment:
Can this be changed to `RebirthDB has dropped the support of Windows`. This will save a commit later.

Reply:
Good idea

Reply:
Good idea